nextflow.config

```diff
@@ -1,155 +1,197 @@
 /*
- * -------------------------------------------------
- *  nf-core/scflow Nextflow config file
- * -------------------------------------------------
- * Default config options for all environments.
- */
-
-// Global default params, used in configs
-params {
-
-  // nf-core-scflow Workflow flags
-  samplesheet = "./refs/SampleSheet.tsv"
-  manifest = "./refs/Manifest.txt"
-  ensembl_mappings = "./src/ensembl-ids/ensembl_mappings.tsv"
-  ctd_folder = "./refs/ctd"
-  celltype_mappings = "./conf/celltype_mappings.tsv"
-  reddim_genes_yml = "./conf/reddim_genes.yml"
-  workDir = "/rds/general/user/$USER/ephemeral/tmp"
-
-  outdir = './results'  
-  publish_dir_mode = 'copy'
-
-  // Boilerplate options
-  name = false
-  email = false
-  email_on_fail = false
-  plaintext_email = false
-  monochrome_logs = false
-  help = false
-  tracedir = "${params.outdir}/pipeline_info"
-  custom_config_version = 'master'
-  custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
-  hostnames = false
-  config_profile_description = false
-  config_profile_contact = false
-  config_profile_url = false
-
-  // Defaults only, expecting to be overwritten
-  max_memory = 128.GB
-  max_cpus = 16
-  max_time = 240.h
+========================================================================================
+    nf-core/scflow Nextflow config file
+========================================================================================
+    Default config options for all compute environments
+----------------------------------------------------------------------------------------
+*/
 
+manifest {
+    name = 'nf-core/scflow'
+    author = 'Dr Combiz Khozoie'
+    homePage = 'https://github.com/nf-core/scflow'
+    description = 'Complete analysis workflow for single-cell/nuclei RNA-sequencing data.'
+    mainScript = 'main.nf'
+    nextflowVersion = '>=21.04.0'
+    version = '0.7.0dev'
 }
 
 // Container slug. Stable releases should specify release tag!
 // Developmental code should specify :dev
-//process.container = 'nfcore/scflow:dev'
-process.container = 'combiz/scflow-docker:0.6.0'
+//process.container = 'combiz/scflow-docker:dev'
+//process.container = 'almurphy/scfdev:latest'
```

Comment:
```suggestion

```

Reply:
Resolved in 05c2a0cfc8338bc3d6edf9fafe119438eb230589